<ui version="4.0" >
 <class>MUCJoin</class>
 <widget class="QDialog" name="MUCJoin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Join Groupchat</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="lb_identity" >
       <property name="text" >
        <string>Identity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_ident" >
       <property name="text" >
        <string>ident</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="TextLabel1_2" >
       <property name="text" >
        <string>Recent:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_recent" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_info" >
     <property name="title" >
      <string>Room information:</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="4" column="0" colspan="2" >
       <widget class="QCheckBox" name="ck_history" >
        <property name="text" >
         <string>Request chat history</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="le_pass" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="TextLabel5" >
        <property name="text" >
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="le_nick" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="TextLabel4" >
        <property name="text" >
         <string>Nickname:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="le_room" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="TextLabel3" >
        <property name="text" >
         <string>Room:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="le_host" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="TextLabel2" >
        <property name="text" >
         <string>Host:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>331</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="Line1" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="BusyWidget" native="1" name="busy" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>81</width>
         <height>29</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_close" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_join" >
       <property name="text" >
        <string>&amp;Join</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>BusyWidget</class>
   <extends>QWidget</extends>
   <header>busywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cb_recent</tabstop>
  <tabstop>le_host</tabstop>
  <tabstop>le_room</tabstop>
  <tabstop>le_nick</tabstop>
  <tabstop>le_pass</tabstop>
  <tabstop>ck_history</tabstop>
  <tabstop>pb_close</tabstop>
  <tabstop>pb_join</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
